dtm0/fop.c

```diff
@@ -21,85 +21,57 @@
 
 
 #define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_DTM0
-#include "lib/trace.h"
-#include "addb2/addb2.h"
-#include "be/dtm0_log.h"
-#include "dtm0/addb2.h"
 #include "dtm0/fop.h"
 #include "dtm0/fop_xc.h"
-#include "dtm0/service.h"
-#include "dtm0/tx_desc.h"
-#include "dtm0/tx_desc_xc.h"
-#include "lib/errno.h"
+#include "dtm0/addb2.h"
+#include "dtm0/drlink.h"       /* m0_dtm0_req_post */
+#include "dtm0/service.h"      /* m0_dtm0_service */
+#include "be/dtm0_log.h"       /* m0_be_dtm0_log_* */
+#include "fop/fom_generic.h"   /* M0_FOPH_* */
 #include "lib/assert.h"
+#include "lib/errno.h"
 #include "lib/memory.h"
-#include "lib/chan.h"
-#include "lib/finject.h"
-#include "lib/time.h"
-#include "lib/misc.h"           /* M0_IN() */
-#include "fop/fop.h"
-#include "fop/fom.h"
-#include "fop/fom_generic.h"
-#include "fop/fop_item_type.h"
-#include "rpc/rpc_opcodes.h"
-#include "rpc/rpc.h"
-#include "rpc/rpclib.h"
-
-static void dtm0_rpc_item_reply_cb(struct m0_rpc_item *item);
-
-/*
-  RPC item operations structures.
- */
-const struct m0_rpc_item_ops dtm0_req_fop_rpc_item_ops = {
-        .rio_replied = dtm0_rpc_item_reply_cb,
-};
+#include "lib/misc.h"          /* M0_IN() */
+#include "lib/trace.h"         /* M0_LOG */
```

Comment:
Usually, this is the next line after 
```
#define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_DTM0
```

Reply:
Thanks, I should have paid attention to that after doing ":sort" on the list of includes. Fixed in https://github.com/Seagate/cortx-motr/pull/802/commits/d3efcfa00717f443705fe3bf9a6e3d27f5d63085